fluentd/datadog_checks/fluentd/fluentd.py

```diff
@@ -51,11 +50,6 @@ def check(self, instance):
                 'fluentd_port:%s' % monitor_agent_port,
             ] + custom_tags
 
-            self.HTTP_CONFIG_REMAPPER = {
-                'headers': {'name': 'headers', 'default': headers(self.agentConfig)},
-                'timeout': {'name': 'timeout', 'default': self.default_timeout},
```

Comment:
Any reason for the removal of `'timeout': {'name': 'timeout', 'default': self.default_timeout},` ?

If removed, `self.default_timeout` declared here would have no effect ? https://github.com/DataDog/integrations-core/blob/3fec2718c71e79602bd8ba31d086bf9baa02c749/fluentd/datadog_checks/fluentd/fluentd.py#L21

Reply:
updated!